packages/dev-utils/src/blockchain_lifecycle.ts

```diff
@@ -2,13 +2,6 @@ import { logUtils } from '@0x/utils';
 import { NodeType, Web3Wrapper } from '@0x/web3-wrapper';
 import * as _ from 'lodash';
 
-// HACK(albrow): 🐉 We have to do this so that debug.setHead works correctly.
```

Comment:
I finally figured out why we needed this hack in the first place. The __genesis.json__ file allows you to configure the block number for certain hard forks (e.g. the DAO hard fork or the Byzantium update). We had some of these configured to block number 1, 2, 3, and 4. I updated the __genesis.json__ file to set all these block numbers to 0 so we no longer need to mine dummy blocks at the start of every test run.